.github/workflows/keycloak-multi-site-health-check.yml

```diff
@@ -59,6 +59,12 @@ jobs:
         with:
           project: ${{ inputs.project }}
 
+      - id: get-keycloak-site-a-url
```

Comment:
My idea is to conditionally run this based on the deployment time from within this workflow, which is probably more flexible.